src/github/entities/user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Base } from 'src/universal/base.entity';
+import { PullRequest } from 'src/webhook/entities/pull-request.entity';
+import { Repo } from 'src/webhook/entities/repository.entity';
+import { Column, Entity, OneToMany } from 'typeorm';
+import { UserInstallation } from './user-installation.entity';
+@Entity()
+export class User extends Base {
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  name: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  phone: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  email: string;
+
+  @Column({ type: 'varchar', nullable: true })
+  password: string;
+
+  @Column({ type: 'varchar', length: 20, unique: true })
+  githubUserId: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true, nullable: true })
+  githubUsername: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  githubAvatarUrl: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  githubAccessToken: string;
+
+  @Column({ type: 'varchar', nullable: true, length: 255 })
+  accessToken: string;
```

Comment:
PullProofAI

Consider adding validation logic for sensitive fields like passwords and access tokens to enhance security.
```typescript
8:   @Column({ type: 'varchar', length: 255, nullable: true })
9:   name: string;

// Add validation for sensitive fields like password and access token
18:   @Column({ type: 'varchar', nullable: true })
19:   password: string;

30:   @Column({ type: 'varchar', length: 255, nullable: true })
31:   githubAccessToken: string;

// Add validation logic for access tokens
```


<!-- This is an auto-generated comment by PullProofAI -->